rendermodules/fusion/fuse_stacks.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+'''
+fuse overlapping render stacks registered by an Affine Homography transform
+    similar to Parallel Elastic Alignment
+'''
+import copy
+
+import renderapi
+from six import viewkeys
+
+from ..module.render_module import RenderModule
+from .schemas import FuseStacksParameters, FuseStacksOutput
+
+example_parameters = {
+    "render": {
+        "host": "em-131fs",
+        "port": 8080,
+        "owner": "testuser",
+        "project": "test",
+        "client_scripts": ""
+    },
+    "stacks": {
+        "stack": "PARENTSTACK",
+        "transform": {
+            "className": "mpicbg.trakem2.transform.AffineModel2D",
+            "dataString": "1.0 0.0 0.0 1.0 0.0 0.0",
+            "type": "leaf"},
+        "children": [{"stack": "CHILDSTACK1",
+                      "transform": {
+                          "className": "mpicbg.trakem2.transform.AffineModel2D",
+                          "dataString": "1.0 0.0 0.0 1.0 0.0 0.0",
+                          "type": "leaf"},
+                      "children": [{"stack": "CHILDSTACK2",
+                                    "transform": {
+                                        "className": "mpicbg.trakem2.transform.AffineModel2D",
+                                        "dataString": "1.0 0.0 0.0 1.0 0.0 0.0",
+                                        "type": "leaf"},
+                                    "children": []}]}]},
+    "pool_size": 12,
+    "interpolate_transforms": True,  # False is notimplemented
+    "output_stack": "FUSEDOUTSTACK",
+    "create_nonoverlapping_zs": True
+}
+
+
+class FuseStacksModule(RenderModule):
+    default_schema = FuseStacksParameters
+    default_output_schema = FuseStacksOutput
+
+    def fusetoparent(self, parent, child, transform=None,
+                     uninterpolated_zs=[]):
+        transform = ([renderapi.transform.AffineModel()]
+                     if transform is None else transform)
+        parent = child if parent is None else parent
+
+        # get overlapping zs
+        parent_zs = set(self.render.run(
+            renderapi.stack.get_z_values_for_stack, parent))
+        child_zs = set(self.render.run(
+            renderapi.stack.get_z_values_for_stack, child))
+        z_intersection = sorted(parent_zs.intersection(child_zs))
+
+        resolved_tspecs = []
+        all_resolved_tform = []
+        for z in uninterpolated_zs:
+            section_tiles = []
+            resolved = renderapi.resolvedtiles.get_resolved_tiles_from_z(
+                child, z, render=self.render)
+            all_resolved_tform += resolved.transforms
+            for ts in resolved.tilespecs:
+                tsc = copy.deepcopy(ts)

```

Comment:
why is this necessary?

Reply:
The copy operation is not strictly necessary -- I had been trying to keep the objects distinct for debugging purposes, but it shouldn't really change anything if we don't stop inside the iteration.